cohort-engine/src/main/java/com/ibm/cohort/engine/CqlEngineWrapper.java

```diff
@@ -50,7 +50,7 @@
  */
 public class CqlEngineWrapper {
 
-	public static final List<String> SUPPORTED_MODELS = Arrays.asList("http://hl7.org/fhir",
+	protected static final List<String> SUPPORTED_MODELS = Arrays.asList("http://hl7.org/fhir",
```

Comment:
> Make this member "protected". 

> Mutable static members of classes and enumerations which are accessed directly, rather than through getters and setters, should be protected to the degree possible. That can be done by reducing visibility or making the field final if appropriate.